prod/westeurope/internal/api/apim/api_management_api_public_r3/terragrunt.hcl

```diff
@@ -0,0 +1,39 @@
+dependency "api_management" {
+  config_path = "../api_management"
+}
+
+dependency "api_management_product_io-public-api" {
+  config_path = "../api_management_product_io-public-api"
+}
+
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_api_management_api?ref=v2.0.34"
+}
+
+inputs = {
+  name                  = "io-public-api-r3"
+  resource_group_name   = dependency.resource_group.outputs.resource_name
+  api_management_name   = dependency.api_management.outputs.name
+  revision              = "1"
+  display_name          = "IO PUBLIC API R3"
+  description           = "PUBLIC API for IO platform."
+  host                  = "api.io.italia.it"
+  path                  = "public"
```

Comment:
@gunzip ditto ?

Reply:
public is right afaik

cc @francescopersico